os.json

```diff
@@ -25,9 +25,8 @@
   "/etc/linuxppc-release" : ["Linux-PPC"],
   "/etc/mageia-release" : ["Mageia"],
   "/etc/mandriva-release" : ["Mandriva Linux", "Mandrake Linux"],
-  "/etc/mandrake-release" : ["Mandriva Linux", "Mandrake Linux"],
+  "/etc/mandrake-release" : ["Mandrake", "Mandriva Linux", "Mandrake Linux"],
```

Comment:
It looks like we've duplicated `Mandrake` here and changed the order of detection would could break `Mandriva`. 

Could we merge `Mandrake Linux` and `Mandrake` and keep it at the end of the array?

Reply:
Done.